data.sql

```diff
@@ -3,4 +3,14 @@
 INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, TRUE, 10.23),
 (2, 'Gabumon', '2018-11-15', 2, TRUE, 8),
 (3, 'Pikachu', '2021-01-07', 1, FALSE, 15.04),
-(4, 'Devimon', '2017-05-12', 5, TRUE, 11);
\ No newline at end of file
+(4, 'Devimon', '2017-05-12', 5, TRUE, 11);
+
+-- TABLE update
+
+INSERT INTO animals VALUES (5, 'Charmander', '2020-02-08', 0, FALSE, -11),
+(6, 'Plantmon', '2021-11-15', 2, TRUE, -5.7),
+(7, 'Squirtle', '1993-6-02', 3, FALSE, -12.13),
+(8, 'Angemon', '2005-06-12', 1, TRUE, -45),
+(9, 'Boarmon', '2005-06-07', 7, TRUE, 20.4);
+(10, 'Blossom', '1998-10-13', 3, TRUE, -17),
```

Comment:
- [OPTIONAL] Kindly make sure you really understand this 👇🏻 
  - Insert the following data: Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times. Let's change the `weight_kg` -17 value, Your code should look like:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Blossom', '1998/10/13', '3', 'true', '17.0');
```

Reply:
Okay, thanks